### PR TITLE
Add Python 3.5 installs to appveyor version matrix

### LINKED ISSUE #174: appveyor.yml missing Python 3.5
The [`appveyor.yml` example](https://github.com/pypa/python-packaging-user-guide/blob/master/source/code/appveyor.yml) at is missing Python 3.5

https://packaging.python.org/en/latest/appveyor/

### PR DESCRIPTION
**Pull request by hickford**
Fixes #174

### CODE REVIEW

File: source/code/appveyor.yml [hunk 1]
```diff
@@ -38,6 +42,11 @@ environment:
       PYTHON_ARCH: "64"
       WINDOWS_SDK_VERSION: "v7.1"
 
+    - PYTHON: "C:\\Python35-x64"
+      PYTHON_VERSION: "3.5.x" # currently 3.5.0
+      PYTHON_ARCH: "64"
+      WINDOWS_SDK_VERSION: "v7.1"
```
Comment (hickford) [inline source/code/appveyor.yml]: Is v7.1 correct?

Comment (ionelmc) [inline source/code/appveyor.yml]: Windows SDK is not necessary for 3.5

Comment (pfmoore) [inline source/code/appveyor.yml]: There have been major changes to this section in master - see https://raw.githubusercontent.com/pypa/python-packaging-user-guide/master/source/code/appveyor.yml and https://packaging.python.org/en/latest/appveyor/#appveyor-yml. The patch is no longer needed as 3.5 is included (in a much simpler form that doesn't need to specify the SDK version in this form anyway).

Comment (xavfernandez) [inline source/code/appveyor.yml]: So maybe this PR should be closed ?

**Comment by hickford** [comment]
Issue https://github.com/pypa/python-packaging-user-guide/issues/174 fixed separately by https://github.com/pypa/python-packaging-user-guide/commit/94fcc0f1d58c628f1bb4213b2033f37ba0db0372#diff-d477b16812c9d7eb4f92bd98c24212d5R13

**Pull request closed without merging**